.cirrus.yml

```diff
@@ -132,10 +132,11 @@ task:
     disk: 50
   env:
     # Cirrus is somehow not picking up the environment variables set in the VM image.
-    PATH: "c:/depot_tools;c:/MinGit/cmd;$PATH"
+    PATH: "c:/depot_tools;c:/MinGit/cmd;c:/flutter/engine/src/out/host_debug_unopt/dart-sdk/bin/;$PATH"
```

Comment:
Can you move this under `build_and_test_web_windows_edge`? `host_debug_unopt` is not available until we build it. Also, this environment is shared across all Windows tasks. Perhaps other tasks want to use a different version of dart (e.g. `host_debug` instead of `host_debug_unopt`).

Reply:
yes I know however as noted in the comments # Cirrus is somehow not picking up the environment variables set in the VM image.

So it is not possible to set a variable other than here.